### PR TITLE
chore: Create Project Resources

### PR DESCRIPTION
**Pull request by hmariset**
Expand Maintainer and Admin roles to create project resources.

### CODE REVIEW

File: deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml [hunk 1]
```diff
@@ -212,6 +212,14 @@ objects:
     - update
     - patch
     - delete
+  - apiGroups:
+    - projctl.konflux.dev
+    resources:
+    - Project
+    - ProjectDevelopmentStreamTemplate
+    - ProjectDevelopmentStream
```
Comment (alexeykazakov) [inline deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml:220]: The resource names should be all lowercase and in a plural form.
```suggestion
    - projects
    - projectdevelopmentstreamtemplates
    - projectdevelopmentstreams
```

File: deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml [hunk 2]
```diff
@@ -212,6 +212,14 @@ objects:
...
+    verbs:
+    - create
```
Comment (alexeykazakov) [inline deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml:222]: Only create? No "list", "get", "delete", etc?

File: deploy/templates/nstemplatetiers/appstudio/spacerole_maintainer.yaml [hunk 3]
```diff
@@ -170,6 +170,14 @@ objects:
     - get
     - list
     - watch
+  - apiGroups:
+    - projctl.konflux.dev
+    resources:
+    - Project
+    - ProjectDevelopmentStreamTemplate
+    - ProjectDevelopmentStream
```
Comment (alexeykazakov) [inline deploy/templates/nstemplatetiers/appstudio/spacerole_maintainer.yaml:178]: Same here.

**Review by alexeykazakov — COMMENTED** [review]
Thank you for the PR but we would need to get an approval from someone from the Konflux Workspace team to get it merged.

Also please create a paired PR for the e2e-test repo. The Konflux Workspace folks can help you with this if needed.

File: deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml (hunk 1 above)
Comment (ifireball) [reply deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml:220]: Since I cant push to @hmariset's fork Created PR #1004 to fix these issues

**Comment by ifireball** [comment]
> Thank you for the PR but we would need to get an approval from someone from the Konflux Workspace team to get it merged.

Are they added automatically as reviewers?

> Also please create a paired PR for the e2e-test repo. The Konflux Workspace folks can help you with this if needed.

Will that be a blocker for merging?

**Comment by mfrancisc** [comment]
Can we close this in favor of https://github.com/codeready-toolchain/host-operator/pull/1004 ?

**Comment by ifireball** [comment]
@mfrancisc 

> Can we close this in favor of #1004 ?

We already made more progress there I think, so yes.

**Pull request closed without merging**